monai/deploy/operators/monai_bundle_inference_operator.py

```diff
@@ -709,18 +723,20 @@ def _convert_from_image_dicom_source(self, img: Image) -> Tuple[np.ndarray, Dict
                 img_meta_dict["depth_pixel_spacing"],
             ]
         )
-        meta_dict["original_affine"] = np.asarray(img_meta_dict.get("nifti_affine_transform", None))
-        meta_dict["affine"] = meta_dict["original_affine"]
+        # Use defines MetaKeys directly
```

Comment:
I don't quite understand this line:
```py
meta_dict = {key: img_meta_dict[key] for key in img_meta_dict.keys()}
```
Maybe just `meta_dict = dict(img_meta_dict)`?

Thanks.

Reply:
Good catch. I think it was meant for deepcopy. Changed accordingly with copy.deepcopy to make it clear.